### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jekwatt**
* Values in column 'collection' are derived from the merge_name
* Query filter for data QC is set based on the production metrics cutoff and the QC Results (PASS or FAIL) get generated in column 'results'
* Tracking weeks is normal priority and will be fixed in another branch

### CODE REVIEW

File: report_merged_qc_results.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def load_merge_report(recent_merge_report):
     # extract abbrev from merge_name
     cid = rtm_sub['merge_name'].str.split('_', n=5, expand=True)[2]
     # add default value using defaultdict
-    d2 = defaultdict(lambda: "NaN")
+    d2 = defaultdict(lambda: "None")
```
Comment (walkerh) [inline report_merged_qc_results.py]: Why `"None"` instead of `None`?

**Review by walkerh — COMMENTED** [review]
Please explain the use of `"None"` instead of `None`.

File: report_merged_qc_results.py (hunk 1 above)
Comment (jekwatt) [reply report_merged_qc_results.py]: It should be None.

Comment (walkerh) [reply report_merged_qc_results.py]: I see you have made that change.

**Review by walkerh — APPROVED** [review]
I'm happy with this.

**Pull request merged**